libraries/classes/User.php

```diff
@@ -133,4 +133,59 @@ public static function isLoggedIn()
       return false;
     }
   }
+
+  public static function isOnline($uid,$token,$address)
+  {
+    $db = new Factory;
+    $mysql = $db->getDBO();
+
+    $res = $mysql->query("SELECT uid FROM user_keys WHERE uid=$uid");
```

Comment:
`SELECT * FROM user_keys WHERE uid=$uid AND ip=$address`
if result is equal to 1 (num_rows) then 
UPDATE `UPDATE user_keys SET uid=$uid,token='$token',isLoggedin=1 WHERE uid = $uid AND ip=$address`
ELSE
INSERT

Its great if we store different IP where user is tried to login